snapmodel_policy.go

```diff
@@ -38,7 +38,16 @@ func computeSnapSystemEpochDigest(alg tpm2.HashAlgorithmId, epoch uint32) tpm2.D
 	return h.Sum(nil)
 }
 
-func computeSnapModelDigest(alg tpm2.HashAlgorithmId, model *asserts.Model) (tpm2.Digest, error) {
+// Model exposes the details of a device model that are to be measured.
+type Model interface {
```

Comment:
As this is in the secboot namespace, it might be worth prefixing this with "Snap" (so, SnapModel) to match the other parts of the API (MeasureSnapModelToTPM, SnapModelProfileParams etc).